prov/shm/src/smr_init.c

```diff
@@ -96,6 +96,9 @@ static int smr_getinfo(uint32_t version, const char *node, const char *service,
 		if (fast_rma) {
 			cur->domain_attr->mr_mode = FI_MR_VIRT_ADDR;
 			cur->tx_attr->msg_order = FI_ORDER_SAS;
+			cur->ep_attr->max_order_raw_size = 0;
+			cur->ep_attr->max_order_waw_size = 0;
+			cur->ep_attr->max_order_war_size = 0;
```

Comment:
The WAW ordering should be okay for any transfers less than the inject size.  That is, 2 writes that are both <= inject size should be written in order.

RAW is ordered in all situations.  Any size read that follows any size write will occur after the write data has been written.

WAR order is not guaranteed in the fast path.